client/src/pages/HomePage.js

```diff
@@ -69,18 +80,31 @@ const HomePage = () => {
         </Grid>
         <Grid item xs={7} style={{ backgroundColor: "#FAFBFF" }} align="right">
           <Box className={classes.box}>
-            <Typography variant="h3" align="left">
-              My mentions
-            </Typography>
+            <div className="flexbox space-between">
```

Comment:
Material UI has [Flexbox ](https://material-ui.com/system/flexbox/) as well as [Grid](https://material-ui.com/components/grid/#grid)

Reply:
I have now used Flexbox in Material UI